Demos/callback/run_cheese.py

```diff
@@ -1,7 +1,7 @@
 import cheese
 
 def report_cheese(name):
-    print("Found cheese: " + name)
+    print("Found cheese: " + name.decode('utf-8'))
```

Comment:
No, I think the right place to do this is in the `callback()` function. It is responsible for converting the C-ish input to something Python-ish. That includes any data mappings.